migration/sql-files/032-invitation-code.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE invitation ADD COLUMN accept_code VARCHAR NOT NULL UNIQUE;
```

Comment:
Will UNIQUE constrain automatically create an index for accept_code?

Reply:
@sbryzak this constraint is missing in master, right?